lab-configs.yaml

```diff
@@ -122,3 +122,7 @@ labs:
             - baseline
             - boot
             - kselftest
+
+  lab-nxp:
+    lab_type: lava
+    url: 'http://59.144.98.45/RPC2/'
```

Comment:
I think this should really be `https` otherwise some API tokens are going to be sent in clear.

Reply:
Of course with https you probably also need a domain name...

Reply:
Ack.  I've requested an https URL + domain-name from NXP.

Reply:
Also it should be between `lab-mhart` and `lab-pengutronix`.

Reply:
@khilman Any news about a domain name for this lab?

Reply:
No news.  I'll ping NXP.

Reply:
Domain name and HTTPS URL now in place.  Updated.

Reply:
Great news!